app/main.js

```diff
@@ -10,7 +10,11 @@ function createWindow () {
   win = new BrowserWindow({
     width: 1000,
     height: 700,
-    icon: path.join(__dirname, 'assets/images/app-icon.png')
+    icon: path.join(__dirname, 'assets/images/app-icon.png'),
+    webPreferences: {
+      nodeIntegration: true,
```

Comment:
This option should not be enabled, so I fix it to false.
https://www.electronjs.org/docs/tutorial/security#2-do-not-enable-nodejs-integration-for-remote-content

Reply:
fixed 8957334f6b7551e0554b338f26be41966428d6d9